src/clinical-trial/clinical-trial.repository.test.ts

```diff
@@ -69,11 +72,69 @@ describe('clinical trial repository', () => {
         'Try draining rhubarb fritters flavored with bourbon.',
         'RSC'
       ),
-      recruitmentStatus
+      recruitmentStatus,
+      new StudyType()
     )
     expect(expectedClinicalTrial).toStrictEqual(clinicalTrial)
   })
 
+  it('should retrieve a Phase I', async () => {
+    // GIVEN
+    const clinicalTrialModel = ClinicalTrialModelTestingFactory.create({ study_type: new StudyType({ phase: Phase.PHASE_1_a }) })
+    const repository = await createRepository([clinicalTrialModel])
+
+    // WHEN
+    const expectedClinicalTrial = repository.findOne('123')
+
+    // THEN
+    const clinicalTrial = new ClinicalTrial(
+      new Title('Resist, scotty, core!', 'RSC'),
+      new Title('Try draining rhubarb fritters flavored with bourbon.', 'RSC'),
+      RecruitmentStatus.RECRUITING,
+      new StudyType({ phase: 'Human Pharmacology (Phase I)- First administration to humans' })
+    )
+
+    expect(expectedClinicalTrial).toStrictEqual(clinicalTrial)
+  })
+
+  it('should retrieve a Phase II/PhaseIII', async () => {
+    // GIVEN
+    const clinicalTrialModel = ClinicalTrialModelTestingFactory.create({ study_type: new StudyType({ phase: Phase.PHASE_2_3 }) })
+    const repository = await createRepository([clinicalTrialModel])
+
+    // WHEN
+    const expectedClinicalTrial = repository.findOne('123')
+
+    // THEN
+    const clinicalTrial = new ClinicalTrial(
+      new Title('Resist, scotty, core!', 'RSC'),
+      new Title('Try draining rhubarb fritters flavored with bourbon.', 'RSC'),
+      RecruitmentStatus.RECRUITING,
+      new StudyType({ phase: 'Phase II and Phase III (Integrated)' })
+    )
+
+    expect(expectedClinicalTrial).toStrictEqual(clinicalTrial)
+  })
+
+  it('should retrieve a empty phase', async () => {
+    // GIVEN
+    const clinicalTrialModel = ClinicalTrialModelTestingFactory.create({ study_type: new StudyType() })
+    const repository = await createRepository([clinicalTrialModel])
+
+    // WHEN
+    const expectedClinicalTrial = repository.findOne('123')
+
+    // THEN
+    const clinicalTrial = new ClinicalTrial(
+      new Title('Resist, scotty, core!', 'RSC'),
+      new Title('Try draining rhubarb fritters flavored with bourbon.', 'RSC'),
+      RecruitmentStatus.RECRUITING,
+      new StudyType()
+    )
+
+    expect(expectedClinicalTrial).toStrictEqual(clinicalTrial)
+  })
+
```

Comment:
Il manque énormément de test...
Si je supprime les trois quart des study type alors aucun test ne casse.
Réutilise le `it.each` pour gagner du temps.